lib/src/widgets/reorder_flex/drag_target.dart

```diff
@@ -165,7 +165,7 @@ class _ReorderDragTargetState<T extends DragTargetData>
           widget.insertAnimationController,
           widget.deleteAnimationController,
         ) ??
-        Draggable<DragTargetData>(
+        LongPressDraggable<DragTargetData>(
```

Comment:
May I ask why replacing the `Draggable` here

Reply:
Hi, I messed up with the main remote branch in my fork. This local fix is not supposed to be part of this PR. Updated.